src/ray/core_worker/core_worker.h

```diff
@@ -295,6 +304,29 @@ class CoreWorker {
                                const std::vector<std::shared_ptr<Buffer>> &metadatas,
                                std::vector<std::shared_ptr<RayObject>> *return_objects);
 
+  /* Handlers for the worker's gRPC server. These are executed on the io_service_ and post
```

Comment:
We should avoid using `/* */` comments (it prevents you from being able to comment things out with them as they can't be nested)